index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="container columns">
+      <nav id="navbar" class="column is-2">
+        <header>Navigation</header>
+        <a href="#" class="nav-link">JavaScript and Java</a>
+        <a href="#" class="nav-link">JavaScript and Java1</a>
+        <a href="#" class="nav-link">JavaScript and Java2</a>
+        <a href="#" class="nav-link">JavaScript and Java3</a>
+        <a href="#" class="nav-link">JavaScript and Java4</a>
+      </nav>
+      <main id="main-doc">
+        <section
```

Comment:
- [ ] [REQUIRED] Well done on this review comment 🙌🏻, however it is not clearly communicated and can not easily be understood by the person you are reviewing.
> A good comment includes a polite suggestion to an issue they found in code, it is then followed by an explanation why this change should be made.

Reply:
![image](https://user-images.githubusercontent.com/63915024/144900255-b71d9128-e194-45df-a662-6e9cf528e66e.png)

Well done, but you have problems that need to be fixed.!
Please 🙌🏻 consider removing a similar `id`. Please You will be able to do so by using `class` or changing the name of `id`.
That is set `id` against the containing elements themselves instead of the internal component elements, and revise your `a` tags accordingly. Or alternately implement your component such that it takes a **namespace** parameter/attribute when used, or replace each `id` to `class` with the namespace inside of your component.
I suggest you read this [article](https://stackoverflow.com/questions/22059438/html-duplicated-id) and watch this [video](https://youtu.be/ZHvn2BXlwdU) to make it even more fun.

**I believe you can do it 💪! Thanks in advance for the work.😊**

[OPTIONAL] I recommend you use this [website](https://validator.w3.org/) to check your code in advance.